bin/prepare-codesandbox.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { merge } = require( 'lodash' );
+
+const currentConfig = JSON.parse(
+	fs.readFileSync( path.join( __dirname, '../.wp-env.json' ), 'utf-8' )
+);
+
+fs.writeFileSync(
+	path.join( __dirname, '../.wp-env.override.json' ),
+	`{
```

Comment:
Nit: Might be easier to read if we just create the object and use `JSON.stringify(config, null, 2)` here.

Reply:
You're right I'm not sure what I was thinking :P